README.md

```diff
@@ -1,2 +1,52 @@
+# Portfolio Juan Felipe Gomez Tobon
+
+> Software Developer: Proficient with HTML, CSS, Javascript, Ruby on Rails programming languages.
+
+![screenshot](./app_screenshot.png)
+
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://felipeg005.github.io/Portfolio/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps
+
+### Prerequisites
+
+- Browser (Chrome,Firefoxe,IE,Opera,Safari)
+
+### Usage
+
+- Open Html.index file with your browser
+
+
+## Authors
+
+👤 **Author**
```

Comment:
- [ ] [Optional] You can add author name here

Reply:
perfect